README.md

```diff
@@ -1,11 +1,10 @@
-Terraform Provider
+Terraform Provider for Google Cloud Platform
 ==================
 
 - Website: https://www.terraform.io
 - [![Gitter chat](https://badges.gitter.im/hashicorp-terraform/Lobby.png)](https://gitter.im/hashicorp-terraform/Lobby)
 - Mailing list: [Google Groups](http://groups.google.com/group/terraform-tool)
 
-![Terraform](https://rawgithub.com/hashicorp/terraform/master/website/source/assets/images/logo-hashicorp.svg)
```

Comment:
It puts a giant Terraform logo in there (like, comically sized; pretty sure it didn't used to be that big).

It felt out of place given that this isn't the main terraform repo, so I opted to remote it for now.